index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html>
+    
+<head>
+    <title>Html</title>
+    <link rel="stylesheet" href="index.css">
+    
```

Comment:
It is advisable to avoid space like this between tags. Leaving such spaces is not a good practice.